zkevm-circuits/src/evm_circuit/util/constraint_builder.rs

```diff
@@ -779,6 +779,30 @@ impl<'a, F: Field> ConstraintBuilder<'a, F> {
         );
     }
 
+    pub(crate) fn account_access_list_read(
+        &mut self,
+        tx_id: Expression<F>,
+        account_address: Expression<F>,
+        value: Expression<F>,
+        value_prev: Expression<F>,
+    ) {
+        self.rw_lookup(
+            "account access list read",
+            false.expr(),
+            RwTableTag::TxAccessListAccount,
+            RwValues::new(
+                tx_id,
+                account_address,
+                0.expr(),
+                0.expr(),
+                value,
+                value_prev,
+                0.expr(),
+                0.expr(),
+            ),
+        );
+    }
```

Comment:
Since it's a read, `value` and `value_prev` should always be the same.

```suggestion
    pub(crate) fn account_access_list_read(
        &mut self,
        tx_id: Expression<F>,
        account_address: Expression<F>,
        value: Expression<F>,
    ) {
        self.rw_lookup(
            "account access list read",
            false.expr(),
            RwTableTag::TxAccessListAccount,
            RwValues::new(
                tx_id,
                account_address,
                0.expr(),
                0.expr(),
                value.clone(),
                value,
                0.expr(),
                0.expr(),
            ),
        );
    }
```